src/Payload.h

```diff
@@ -356,6 +367,9 @@ class OmnmPayloadImpl {
 
     // Meta data for the payload
     omnmHeader    mHeader;
+
+    // Offset information for all fields
+    std::vector<fieldHint>   mFieldHints;
```

Comment:
I'm skeptical about potential performance issues here and what it's used for. Would need to test.

Reply:
Please do -- in our testing the difference was orders of magnitude.